src/go/configgenerator/cluster_generator.go

```diff
@@ -196,6 +198,24 @@ func makeJwtProviderClusters(serviceInfo *sc.ServiceInfo) ([]*v2pb.Cluster, erro
 	return providerClusters, nil
 }
 
+func parseDnsResolverAddress(address string) (string, uint32, error) {
+	arr := strings.Split(address, ":")
```

Comment:
can we re-use util.ParseURL() 

or 
create a new function ParseHostPort() so the code can be shared

Reply:
This method is so hard to share so still put in dns_resolver file